From e1f1deb06d9168a95a381a2236e1d8c693d3d229 Mon Sep 17 00:00:00 2001
From: Luciano Chavez <lnx1138@linux.ibm.com>
Date: Wed, 24 Aug 2022 21:17:54 -0500
Subject: [PATCH] lsslot: Explicity declare that lmb_address be displayed in
 hexadecimal

Upstream: Expected v1.3.11
Git-commit: e1f1deb06d9168a95a381a2236e1d8c693d3d229

A printf statement used is lsslot.c was specifying the macro PRIu64 to
display the lmb_address. Depending on the compilation, this would
either display as a hexadecimal or decimal value.

This patch replaces PRIu64 with PRIx64 to explicitly declare to print
the value as hexadecimal as that was is normally expected of an address.

Signed-off-by: Luciano Chavez <lnx1138@linux.ibm.com>
Signed-off-by: Tyrel Datwyler <tyreld@linux.ibm.com>
---
 src/drmgr/lsslot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/drmgr/lsslot.c b/src/drmgr/lsslot.c
index 7ea0f8b..87f876e 100644
--- a/src/drmgr/lsslot.c
+++ b/src/drmgr/lsslot.c
@@ -756,7 +756,7 @@ int print_drconf_mem(struct lmb_list_head *lmb_list)
 		printf("%s: %s\n", lmb->drc_name,
 		       lmb->is_owned ? "" : "Not Owned");
 
-		printf("    DRC Index: %x        Address: %"PRIu64"\n",
+		printf("    DRC Index: %x        Address: %"PRIx64"\n",
 		       lmb->drc_index, lmb->lmb_address);
 		printf("    Removable: %s             Associativity: ",
 		       lmb->is_removable ? "Yes" : "No ");
-- 
2.37.3

